app/javascript/shared/helpers/KeyboardHelpers.js

```diff
@@ -127,3 +127,14 @@ export const buildHotKeys = e => {
   hotKeyPattern += key;
   return hotKeyPattern;
 };
+
+export const isActiveElementTypeable = e => {
+  const activeElement = e.target ?? document.activeElement;
+
+  return !!(
+    activeElement?.tagName === 'INPUT' ||
+    activeElement?.tagName === 'NINJA-KEYS' ||
+    activeElement?.tagName === 'TEXTAREA' ||
+    activeElement?.contentEditable === 'true'
+  );
+};
```

Comment:
@scmmishra, Could you please take a look at the `onKeyDownHandler(e)` method in `eventListenerMixins`? Currently, it only handles 'INPUT' and 'TEXTAREA', but we can add the rest there and we can remove this function from here.

Reply:
Sounds good! Will do